internal/lm/fabric_test.go

```diff
@@ -32,7 +32,7 @@ func TestGerenerateDomainUUID(t *testing.T) {
 		{
 			description: "single IP",
 			ips:         []string{"10.130.3.24"},
-			expected:    "4dbd3d31-fbb3-8a40-33bb-bcc0dd7b68b8",
+			expected:    "60ad7226-0130-54d0-b762-2a5385a3a26f",
```

Comment:
@klueska @ArangoGutierrez are we OK with this "breaking" change given that we haven't released this yet.

I think the simplifications in this PR are worth it.

Reply:
yes

Reply:
I've rebased the PR. Requesting another round of review